replica/tablets.cc

```diff
@@ -195,8 +214,12 @@ future<tablet_metadata> read_tablet_metadata(cql3::query_processor& qp) {
                 throw std::runtime_error(format("Too many pending replicas for table {} tablet {}: {}",
                                                 table, current->tid, pending));
             }
+            service::session_id session_id;
+            if (row.has("session")) {
+                session_id = service::session_id(row.get_as<utils::UUID>("session"));
+            }
```

Comment:
Since we set session_id only if the field is present it means that at other times it will default constructable, but  isn't this the same as default_session_id? Can it be dangerous?

Reply:
Yes, it's better to initialize default_session_id to something else.